### DISCUSSION TITLE
How to build to contribute a fix

### DISCUSSION THREAD

**Discussion by IRobot1** [angular-threejs/angular-three#8] [Ideas]
Can you add some documentation how the various projects are built and work together? 

The end goal is how to fix a bug in each of the repositories in order to submit a pull request.  

Best not to assume everyone has your level of understanding so be explicit about things that might be obvious to you.

**Comment by IRobot1** [comment]
Can you add an example app for postprocessing so its simpler to test a fix?

**Reply by nartc** [reply]
I'll add an example app to postprocessing.

**Reply by IRobot1** [reply]
Just wondering when you'll be able to add this.

**Reply by nartc** [reply]
I can add an app to postprocessing now. I'm considering grouping everything back into a single monorepo again but waiting on v2 release with IDE support to do so.

**Reply by nartc** [reply]
@IRobot1 added example to postprocessing

**Comment by nartc** [comment]
Thanks for raising this. I'll add a CONTRIBUTING guide for everything
